index.html

```diff
@@ -38,12 +38,16 @@
   <script type="riot/tag" src="tags/detail-app-container.tag"></script>
   <script type="riot/tag" src="tags/edit-app-container.tag"></script>
 
-  <script src="vendor/jquery-2.2.4.min.js"></script>
+  <script>
+    // https://github.com/electron/electron/issues/254
+    window.$ = window.jQuery = require('./vendor/jquery-2.2.4.min.js');
```

Comment:
@gr2m Thank you so much for the wonderful example code. It's really helpful ;) 
I have a few questions. Does this code make JQuery an global object or not ? If it is, why do we need to make it global object here?